packages/evm/contracts/validators/SignerValidator.sol

```diff
@@ -141,7 +142,7 @@ library IncentiveBits {
             // toggle the value that was stored inline on stack with xor
             let updatedStorageValue := xor(sload(storageSlot), shl(incentive, 1))
             // isolate the toggled bit and see if it's been unset back to zero
-            let alreadySet := xor(1, shr(incentive, updatedStorageValue))
+            let alreadySet := xor(1, and(1, shr(incentive, updatedStorageValue)))
```

Comment:
tfw no `NAND`

Reply:
Legend has it you can build a whole computer with those things